core/src/main/java/org/kohsuke/stapler/framework/io/LargeText.java

```diff
@@ -63,7 +63,7 @@ public class LargeText {
     /**
      * Represents the data source of this text.
      */
-    private interface Source {
+    public interface Source {
         Session open() throws IOException;
```

Comment:
can we document these methods now that the interface is public?

Reply:
Done in 50ce95ca28a81a072262f166a08cf85799045f83